erpnext/accounts/doctype/purchase_invoice/purchase_invoice.py

```diff
@@ -323,10 +323,11 @@ def validate_cash(self):
 
 	def create_remarks(self):
 		if not self.remarks:
-			if self.bill_no and self.bill_date:
-				self.remarks = _("Against Supplier Invoice {0} dated {1}").format(
-					self.bill_no, formatdate(self.bill_date)
-				)
+			if self.bill_no:
+				self.remarks = _("Against Supplier Invoice {0}").format(self.bill_no)
```

Comment:
@ruthra-kumar 
I can patch this if required. Update remarks where it's not set. 

Thoughts?

Reply:
Update with what value?

Reply:
Patch with update remarks,
- Where remarks = No Remarks
- And Bill Number is set.

Reply:
I've added a patch.